src/Kernel/Object.class.st

```diff
@@ -2101,6 +2101,13 @@ Object >> writeSlotNamed: aName value: anObject [
 	^(self class slotNamed: aName) write: anObject to: self
 ]
 
+{ #category : #'process termination handling' }
+Object >> yield [ 
```

Comment:
I like the idea in principal, but adding methods to Object potentially constrains and clashes with application domain.
For example, a financial application might want to implement #yield...
https://www.investopedia.com/terms/y/yield.asp

And there are many others...
https://www.merriam-webster.com/dictionary/yield

Can we make this method more specific, i.e. #yieldProcessor ?
```suggestion
Object >> yieldProcessor [
```